src/components/plots/MultiViewGrid.jsx

```diff
@@ -1,39 +1,137 @@
-import React, { useEffect, useState, useRef } from 'react';
+import React, {
+  useEffect, useState, useRef, useCallback,
+} from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 
 import _ from 'lodash';
 import { Col, Row, Space } from 'antd';
 import { getPlotConfigs, getCellSets } from 'redux/selectors';
 import { loadCellSets } from 'redux/actions/cellSets';
+import {
+  updatePlotConfig,
+  savePlotConfig,
+} from 'redux/actions/componentConfig/index';
 
 import PlatformError from 'components/PlatformError';
+import loadConditionalComponentConfig from 'redux/actions/componentConfig/loadConditionalComponentConfig';
+import Loader from 'components/Loader';
+import { loadPaginatedGeneProperties, loadGeneExpression } from 'redux/actions/genes';
+import { plotTypes, plotUuids } from 'utils/constants';
+
+const PROPERTIES = ['dispersions'];
+
+const tableState = {
+  pagination: {
+    current: 1, pageSize: 1000000, showSizeChanger: true,
+  },
+  geneNamesFilter: null,
+  sorter: { field: PROPERTIES[0], columnKey: PROPERTIES[0], order: 'descend' },
+};
+
+const multiViewType = plotTypes.MULTI_VIEW_PLOT;
 
 const MultiViewGrid = (props) => {
   const {
     experimentId,
     renderPlot,
-    multiViewUuid,
     updateAllWithChanges,
+    plotType,
+    plotUuid,
   } = props;
-
+  const multiViewUuid = plotUuids.getMultiPlotUuid(plotType);
   const dispatch = useDispatch();
 
   const cellSets = useSelector(getCellSets());
 
-  const multiViewConfig = useSelector((state) => state.componentConfig[multiViewUuid].config);
-  const plotConfigs = useSelector(getPlotConfigs(multiViewConfig.plotUuids));
+  const multiViewConfig = useSelector((state) => state.componentConfig[multiViewUuid]?.config);
+  const plotConfigs = useSelector(getPlotConfigs(multiViewConfig?.plotUuids));
+  const multiViewPlotUuids = multiViewConfig?.plotUuids;
```

Comment:
We have `multiViewUuid`, `multiViewPlotUuids`, `firstPlotUuid` -- it gets confusing. Specifically what is the difference between `multiViewUuid` and `multiViewPlotUuids` and do we need a variable for each of them?

Reply:
multiViewUuid = `multiView-violin`

MultiplotUuids = plotUuids  under` multiView-violin` so 
`multiView-violin: plotUuids[violin-0, violin-1, violin-2....]`

Reply:
you mean `multiViewPlotUuids`, not `MultiplotUuids`, right? Variable `MultiplotUuids` doesn't seem to exist.

Reply:
Yes i meant multiViewPlotUuids